io_scene_godot/converters/multimesh.py

```diff
@@ -0,0 +1,177 @@
+import bpy
+import mathutils
+import math
+import re
+from ..structures import (
+    Array, NodeTemplate, InternalResource, Map, gamma_correct,fix_matrix)
+from .mesh import ArrayMeshResourceExporter
+
+def export_multimesh_node(escn_file, export_settings,
+                           obj, parent_gd_node):
+    """Export a blender object with parent_bone to a BoneAttachment"""
+    context = bpy.context
+    dg = context.evaluated_depsgraph_get()
+    ob = context.object.evaluated_get(dg)
+
+    ps = ob.particle_systems.active
+    if ps.settings.instance_collection and ps.settings.instance_collection.all_objects[0]:
+        instance_object=ps.settings.instance_collection.all_objects[0]
+    elif ps.settings.instance_object:
+        instance_object=ps.settings.instance_object
+
+    multimeshnode = NodeTemplate(re.sub('[^a-zA-Z0-9]','',ps.name) + 'MultiMesh',
+                                   'MultiMeshInstance', parent_gd_node)
+
+    # Export instance mesh resource first
+    instance_mesh_exporter = ArrayMeshResourceExporter(instance_object)
+    # armature_obj = None
+    # if "ARMATURE" in export_settings['object_types']:
+    #     armature_obj = get_modifier_armature(obj)
+    #     if armature_obj:
+    #         instance_mesh_exporter.init_mesh_bones_data(armature_obj, export_settings)
+    #         # set armature to REST so current pose does not affect converted
+    #         # meshes.
+    #         armature_pose_position = armature_obj.data.pose_position
+    #         armature_obj.data.pose_position = "REST"
+    mesh_id = instance_mesh_exporter.export_mesh(escn_file, export_settings)
+
+    multimeshExporter = MultiMeshResourceExporter(obj,mesh_id,ps)
+
+
+    multimeshid = multimeshExporter.export_multimesh(escn_file,export_settings,ps.name)
+
+    multimeshnode['multimesh'] = 'SubResource(%d)' % multimeshid

```

Comment:
prefer `format` to `%` for forward-compatibility.

Reply:
Is `%` actually deprecated and going to be removed in a future version? [pyformat.info](https://pyformat.info/) doesn't mention that.

Reply:
This was surprisingly hard to dig up: https://docs.python.org/2/library/stdtypes.html#str.format

> This method of string formatting is the new standard in Python 3, and should be preferred to the % formatting described in String Formatting Operations in new code.

And https://docs.python.org/3/library/stdtypes.html#printf-style-string-formatting: 

> The formatting operations described here exhibit a variety of quirks that lead to a number of common errors (such as failing to display tuples and dictionaries correctly). Using the newer formatted string literals, the str.format() interface, or template strings may help avoid these errors. Each of these alternatives provides their own trade-offs and benefits of simplicity, flexibility, and/or extensibility.

So it seems generally warned against, but probably nothing terrible will happen if you decide to use it 🤷